PingProtect/PingProtect/Sources/PIInitParams.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  PIInitParams.swift
+//  PingProtect
+//
+//  Copyright (c) 2024 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+import PingOneSignals
+
+/// Parameters for starting PIProtect SDK
+public struct PIInitParams {
```

Comment:
do we wanna follow the same rule for ours as well ?

    @Nullable
    protected String envId;
    @Nullable
    private List<String> deviceAttributesToIgnore;
    @Nullable
    private String customHost;
    private boolean consoleLogEnabled = false;
    private boolean lazyMetadata = false;
    private boolean behavioralDataCollection = true;

Reply:
Updated